lang/common/proto/evaluator_shim.proto

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+option java_package = "org.apache.reef.proto";
+option java_outer_classname = "EvaluatorShimProtocol";
```

Comment:
I'm not sure "EvaluatorShim" is really a descriptive enough name. An incoming person would be curious what exactly your trying to Shim and why.

Reply:
Any suggestions? I see Mesos runtime calls their shim REEFExecutor. I'm not particular about naming.

Reply:
I just think of this functionality more as an EvaluatorBootstrap or NodeLauncher than a traditional shim, but maybe that's just me.. We can think about the naming for a later PR.